index.html

```diff
@@ -16,39 +16,39 @@ <h1 class="text-center" id="workWithUs">Work with Us</h1>
       </p>
     </div>
     <div class="padded-content workwithusContent">
-      <div class="row workwithusIconsRow col-md-10">
-        <div class="col-md-3 col-xxssm-6">
+      <div class="row workwithusIconsRow iconContainer">
```

Comment:
Sorry if this wasn't clear, but the `row` class also comes from Bootstrap (I think we add some additional styles to it in `main.scss`). 
Can we update the `row` class to include the styles that Bootstrap gives us?

Reply:
Ignore this! I just noticed we have a separate ticket to handle the `row` class.